draft-ietf-rtcweb-jsep.xml

```diff
@@ -3751,13 +3767,14 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
 
           <t>If the ICE ufrag or password changed from the previous
           remote description:
-          <xref target="RFC5245" />.
+          <xref target="RFC8445" />.
```

Comment:
This line is a typo and should just be removed.

Reply:
Done